### PR TITLE
Add multi how-to

### PR DESCRIPTION
**Pull request by msaraiva**

### CODE REVIEW

File: guides/howtos/Composable transactions with Multi.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+This is considerably simpler than the nested transaction approach we have seen earlier. Once all operations are defined in the multi, we can finally call `Repo.transaction`, this time passing the multi:
+
+```elixir
+transfer_money(mary, john, 10)
+|> Ecto.Multi.insert(:transfer, %Transfer{from: mary.id, to: john.id, amount: 10})
+|> Repo.transaction()
+|> case do
+  {:ok, %{transfer: transfer}} ->
+    # Handle success case
+  {:error, name, value, rolled_back_changes} ->
```
Comment (AndrewDryga) [inline guides/howtos/Composable transactions with Multi.md]: I think `rolled_back_changes` is a misleading name. One of Multi steps can contain a function which result is not rolled back along with DB transaction.

Comment (josevalim) [reply guides/howtos/Composable transactions with Multi.md]:
```suggestion
  {:error, name, value, changes_so_far} ->
```

**Comment by josevalim** [comment]
Looks great!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**